README.md

```diff
@@ -170,6 +169,10 @@ Please note for this to work you should create/update user maven settings (typic
      </servers>
      ...
 
+### Installing all projects to the local Maven repository ###
+
+    ./gradlew publishToMavenLocal
```

Comment:
Could we combine this with line #178 into a single command? Maybe `./gradlew -PskipSigning=true publishToMavenLocal` is good enough

Reply:
Good point. I'll still keep the separate line for the single project case, but will add the `-PskipSigning=true` to this one.

Reply:
Pushed this change.